unified-runtime/test/conformance/lit.cfg.py

```diff
@@ -7,5 +7,68 @@
 
 """
 
-# Disable lit testing (for now)
-config.suffixes = []
+import lit.formats
+import subprocess
+from os import path
+
+
+class NopTestFormat(lit.formats.TestFormat):
+    def getTestsForPath(*args, **kwargs):
+        return
+        yield
+
+    def getTestsInDirectory(*args, **kwargs):
+        return
+        yield
+
+
+config.name = "Unified Runtime Conformance"
+config.test_format = lit.formats.GoogleTest("", "-test")
+config.test_source_root = config.binary_dir
+config.test_exec_root = config.binary_dir
+
+if config.using_l0_v2:
+    config.environment["UR_LOADER_USE_LEVEL_ZERO_V2"] = "1"
+else:
+    config.environment["UR_LOADER_USE_LEVEL_ZERO_V2"] = "0"
+
+if "UR_CTS_ALSO_RUN_KNOWN_FAILURES" in os.environ:
+    config.environment["UR_CTS_ALSO_RUN_KNOWN_FAILURES"] = os.environ[
+        "UR_CTS_ALSO_RUN_KNOWN_FAILURES"
+    ]
+
+if "selector" in lit_config.params:
```

Comment:
The use of `in` and `get` with an empty default with multiple branches can be folded into this, which I think is a little more idiomatic python:

```python
default_selector = ':,*'.join(c for c in config.adapters_built if c != "mock")
selector = os.environ.get(
   "ONEAPI_DEVICE_SELECTOR",
   lit_config.params.get("selector", default_selector)
)
config.environment["ONEAPI_DEVICE_SELECTOR"] = selector
if not selector:
    lit_config.warning(
        "No device selector is set (are any adapters enabled?), skipping conformance tests"
    )
```
(Also, please see my other comment about empty selectors below)

Reply:
The use of "chaining defaults" like that looks to me like it would obfuscates the meaning of the code and make it hard to skim through. Multiple ifs are clearer IMO. I'll redesign the default selector logic though.